Topics/Tech_Stacks/Tanstack_Table.md

```diff
@@ -0,0 +1,141 @@
+# How to Build Customizable Tables in React
```

Comment:
You should add a small table of contents to facilitate navigation

Reply:
I'm not sure if this page would need that or not, since it is relatively short. I do see it potentially being quite long but at this current length I'm not sure if it's a good idea or not.